source/payment_flow/index.html.md.erb

```diff
@@ -11,7 +11,7 @@ This section outlines how your service will interact with GOV.UK Pay after you i
 
 There's a different process for [taking Direct Debit payments](/direct_debit/#take-a-direct-debit-payment).
 
-<%= warning_text('This does not apply to your users who use the <a href="https://www.payments.service.gov.uk/govuk-payment-pages/">payment links</a> functionality') %>
+<%= warning_text('This does not apply if you only use <a href="https://www.payments.service.gov.uk/govuk-payment-pages/" target="_blank">payment links</a>.') %>
```

Comment:
this is also a slightly different call than the regular create a payment API.
since we're not really doing on DD at the moment I'd be inclined leave this page as is.

Reply:
ok will change back